app/wiring/AppComponents.scala

```diff
@@ -48,9 +48,15 @@ class AppComponents(context: Context, stage: String) extends BuiltInComponentsFr
     )
   }
 
+  private val twoFactorAuthEnforceGoogleGroup = configuration.get[String]("googleAuth.2faEnforceGroup")
   private val requiredGoogleGroups: Set[String] = configuration.get[String]("googleAuth.requiredGroups").split(',').toSet
```

Comment:
probably should have renamed this to `allowedGroups` as they are not all required any more

Reply:
That is a very good point - it would make it a lot clearer. I'm going to do this in a follow up PR

Reply:
addressed here https://github.com/guardian/support-admin-console/pull/655